Source/ASNetworkImageNode.mm

```diff
@@ -98,12 +100,14 @@ - (instancetype)initWithCache:(id<ASImageCacheProtocol>)cache downloader:(id<ASI
   _downloaderFlags.downloaderImplementsSetPriority = [downloader respondsToSelector:@selector(setPriority:withDownloadIdentifier:)];
   _downloaderFlags.downloaderImplementsAnimatedImage = [downloader respondsToSelector:@selector(animatedImageWithData:)];
   _downloaderFlags.downloaderImplementsCancelWithResume = [downloader respondsToSelector:@selector(cancelImageDownloadWithResumePossibilityForIdentifier:)];
+  _downloaderFlags.downloaderImplementsDownloadWithPriority = [downloader respondsToSelector:@selector(downloadImageWithURL:priority:callbackQueue:downloadProgress:completion:)];
 
   _cacheFlags.cacheSupportsClearing = [cache respondsToSelector:@selector(clearFetchedImageFromCacheWithURL:)];
   _cacheFlags.cacheSupportsSynchronousFetch = [cache respondsToSelector:@selector(synchronouslyFetchedCachedImageWithURL:)];
   
   _shouldCacheImage = YES;
   _shouldRenderProgressImages = YES;
+  _isInImageDownloaderPriorityExperiment = ASActivateExperimentalFeature(ASExperimentalImageDownloaderPriority);
```

Comment:
Let's not cache this 😱  – ASActivateExperimentalFeature is fast enough to be called repeatedly so let's leave it as the source of truth.

Reply:
Ah yeah, my bad. I didn't look into the implementation of `ASActivateExperimentalFeature`.

I assumed that it calls out to the configuration delegate every time. Our experiment manager is known to have measurable perf impacts if it's repeatedly called many times within a frame.

But since `ASActivateExperimentalFeature` calls its delegate only a new experiment is triggered, we don't need to cache this. Gonna update the diff.